modules/ztp-worker-node-preparing-policies.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-additional-worker-policies_{context}"]
+= Applying worker node policies
```

Comment:
I think this module could lose the the secondary headings (`==`) to make it more focused

```suggestion
= Using PolicyGenTemplate CRs to apply worker node policies to worker nodes
```